src/core/lib/ConsiderationEncoder.sol

```diff
@@ -571,8 +574,19 @@ contract ConsiderationEncoder {
         uint256 size,
         uint256 memoryLocationForOrderHashes
     ) {
-        // Get free memory pointer to write calldata to.
-        ptr = getFreeMemoryPointer();
+        unchecked {
+            // Derive offset to pre `OrderFulfilled`'s spent item event data
+            // using base offset & total original recipients.
+            ptr = MemoryPointer.wrap(
```

Comment:
note for myself: The original `eventDataPtr` is @ `OrderFulfilled_baseOffset=200 + BasicOrder_totalOriginalAdditionalRecipients_cdPtr * 0x20`. Here we go to `authorizeOrder_calldata_baseOffset=0x80 + BasicOrder_totalOriginalAdditionalRecipients_cdPtr * 0x20`. So we're 0x180 in front of it, which is what we want.